lib/checks.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+
+function checks::ensure_supported_stack() {
+	local stack="${1}"
+
+	case "${stack}" in
+		scalingo-20 | scalingo-22 | scalingo-24)
+			return 0
+			;;
+		cedar* | scalingo-14 | scalingo-18)
```

Comment:
suggestion: I would keep the Heroku stack names in these conditions. Even though it became really rare, some customers (or even internal app...) may have a Heroku app name in the environment. I can send you private links in DM if needed